pkg/apis/pipeline/options.go

```diff
@@ -18,6 +18,7 @@ package pipeline
 
 // Options holds options passed to the Tekton Pipeline controllers
 // typically via command-line flags.
+// +k8s:deepcopy-gen=false
```

Comment:
Sure!  The code generator is applied to this pkg(apis/pipeline) now to generate deepcopy for internal version structs, so all structs in this pkg will generate deepcopy code. The previous structs don't need to generate deepcopy code, that's the reason we set it to false.

Reply:
Did u mean pkg/apis/pipeline/internalversion? That's another option discussed [here](https://docs.google.com/document/d/1wXQaiay18hlcuxOvl5T3BZiyOFSLN9hsr4rkhOwCz6I/edit#heading=h.xzptrog8pyxf) I think either is fine, if we want to follow the same convention as k8s, we could choose the approach in this pr

Reply:
Interesting, what about putting the existing files to a separate folder? I think it could be better to separate the existing files from the intern versions, but I don't have strong opinion on it.

Reply:
Oh I see, that is also one option